pkg/v1/tkg/test/tkgctl/shared/tkr_cc_misc.go

```diff
@@ -146,7 +147,7 @@ func ValidateTKRsRelatedObjectsAvailability(ctx context.Context, mcProxy *framew
 		tkr := &runv1.TanzuKubernetesRelease{}
 		err := crClient.Get(ctx, client.ObjectKey{Name: expectedTKRs[i]}, tkr)
 		Expect(err).To(BeNil(), fmt.Sprintf("failed to get TKR :%s", expectedTKRs[i]))
-		cbt := getClusterBootstrapTemplate(ctx, crClient, tkr.Name)
+		cbt := getClusterBootstrapTemplate(ctx, crClient, tkr.Name, constants.TKGNamespace)
```

Comment:
Not related to your change, but wondering why this check is in ```tkr_cc_misc.go``` file?!

Reply:
refer to first comment